cypress/e2e/tests/migration/dynamic-report/issues/affectedApplications/sort.test.ts

```diff
@@ -0,0 +1,92 @@
+import { Analysis } from "../../../../../models/migration/applicationinventory/analysis";
+import {
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    validateSortBy,
+} from "../../../../../../utils/utils";
+import { SEC } from "../../../../../types/constants";
+import { BusinessServices } from "../../../../../models/migration/controls/businessservices";
+import * as data from "../../../../../../utils/data_utils";
+import { getRandomWord } from "../../../../../../utils/data_utils";
+import { Issues } from "../../../../../models/migration/dynamic-report/issues/issues";
+
+describe(
+    ["@tier2"],
+    "Validating application sorting in Issues -> affected applications",
+    function () {
+        let applicationsList: Array<Analysis> = [];
+        let businessServiceList: Array<BusinessServices> = [];
+        const sortByList = ["Name", "Business serice", "Effort", "Incidents"];
```

Comment:
Typo : "Business service "

Reply:
Typo is in UI, if I will write here without typo - test will fail